pkg/client/connector/service_proxy.go

```diff
@@ -64,6 +64,15 @@ func (p *mgrProxy) CanConnectAmbassadorCloud(ctx context.Context, arg *empty.Emp
 	}
 	return client.CanConnectAmbassadorCloud(ctx, arg, callOptions...)
 }
+
+func (p *mgrProxy) GetCloudConfig(ctx context.Context, arg *empty.Empty) (*managerrpc.AmbassadorCloudConfig, error) {
+	client, callOptions, err := p.get()
+	if err != nil {
+		return nil, err
+	}
+	return client.GetCloudConfig(ctx, arg, callOptions...)
+}
```

Comment:
Does this ever get called? I know it's implementing the interface.  I almost want to say "just have it always return an error, it should never be called because the client should already have the config."

Reply:
I'll add a comment about investigating it's potential removal.  I think when I did it, I was mostly just seeing that pretty much every function was this way, so I followed the pattern.  Which I should have dug in a little further to actually ensure that it was necessary.